libcxx/test/std/time/time.zone/time.zone.zonedtime/time.zone.zonedtime.members/get_local_time.pass.cpp

```diff
@@ -0,0 +1,133 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+// UNSUPPORTED: no-filesystem, no-localization, no-tzdb
+
+// XFAIL: libcpp-has-no-experimental-tzdb
+// XFAIL: availability-tzdb-missing
+
+// <chrono>
+
+// template<class Duration, class TimeZonePtr = const time_zone*>
+// class zoned_time;
+//
+// local_time<duration> get_local_time() const;
+
+#include <chrono>
+#include <concepts>
+
+#include "../test_offset_time_zone.h"
+
+namespace cr = std::chrono;
+
+static void test_const_member() {
+  {
+    using duration         = cr::nanoseconds;
+    using time_point       = cr::sys_time<duration>;
+    using local_time_point = cr::local_time<duration>;
+    cr::zoned_time<duration> zt{"Etc/GMT+1", time_point{duration{42}}};
```

Comment:
This test seems out of place, since the function name suggests that we're only testing const-qualification in it.

Reply:
I adjusted the name of the function to better show its purpose.